doc/contribute/documentation/guidelines.rst

```diff
@@ -525,121 +706,401 @@ or if you'd like to add an image caption, use::
 
 The file name specified is relative to the document source file,
 and we recommend putting images into an ``images`` folder where the document
-source is found.  The usual image formats handled by a web browser are
-supported: JPEG, PNG, GIF, and SVG.  Keep the image size only as large
-as needed, generally at least 500 px wide but no more than 1000 px, and
-no more than 250 KB unless a particularly large image is needed for
-clarity.
+source is found.
 
-Tabs, spaces, and indenting
-***************************
+The usual image formats handled by a web browser are supported: WebP, PNG, GIF,
+JPEG, and SVG.
 
-Indenting is significant in reST file content, and using spaces is
-preferred.  Extra indenting can (unintentionally) change the way content
-is rendered too.  For lists and directives, indent the content text to
-the first non-white space in the preceding line.  For example::
+Keep the image size only as large as needed, generally at least 500 px wide but
+no more than 1000 px, and no more than 100 KB unless a particularly large image
+is needed for clarity.
 
-   * List item that spans multiple lines of text
-     showing where to indent the continuation line.
+Recommended image formats based on content
+------------------------------------------
 
-   1. And for numbered list items, the continuation
-      line should align with the text of the line above.
+* **Screenshots**: WebP or PNG.
```

Comment:
Would it make sense to also refer to some compression tools such as https://tinypng.com/ or similar? In most cases at least 25-50% of the size can be reduced without losing any noticeable quality, but can reduce size of the documentation significantly.

Reply:
yes I was thinking of this but honestly wanted to get a first batch of updates out as I feel like I'm already pushing too much in one single PR (and commit :)).